WordPress/Classes/ViewRelated/Me/MyDomains/PurchaseDomainView.swift

```diff
@@ -0,0 +1,158 @@
+import SwiftUI
+
+struct PurchaseDomainView: View {
```

Comment:
[Suggestion] The name of this `struct` gives the impression that this is screen where users actually purchase domains, but we know the screen isn't about that. I'm suggesting to change the name of this view, here are some alternatives:
- DomainPurchaseChoicesView
- HowToUseDomainView
- SiteOrDomainView ( web folks use this naming )

Reply:
Fair point. I think I like your first two suggestions more. Do you have preference between those two?

Reply:
Sorry, I just saw your comment. I saw you changed the name to `DomainPurchaseChoicesView`. That was my preference too.